src/common/config.cpp

```diff
@@ -619,11 +621,15 @@ void Config::GenData() {
   for (size_t i = 0; i < this->frame_.NumULSyms(); i++) {
     for (size_t j = 0;
          j < this->ldpc_config_.NumBlocksInSymbol() * this->ue_ant_num_; j++) {
+      for (size_t k = 0; k < bytes_per_block; k++)
```

Comment:
```suggestion
      for (size_t k = 0; k < bytes_per_block; k++) {
```

Reply:
Should we make this loop into a function call into Wlan function set?  I think it happens in a few places.